tdp/cli/commands/plan/import_file.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2022 TOSIT.IO
+# SPDX-License-Identifier: Apache-2.0
+
+from __future__ import annotations
+
+import logging
+from typing import TYPE_CHECKING
+
+import click
+
+from tdp.cli.queries import get_planned_deployment
+from tdp.cli.session import get_session
+from tdp.cli.utils import collections, database_dsn, parse_file
+from tdp.core.models.deployment_model import DeploymentModel
+
+if TYPE_CHECKING:
+    from tdp.core.collections import Collections
+
+logger = logging.getLogger("tdp").getChild("edit")
```

Comment:
You should use:

```suggestion
logger = logging.getLogger(__name__)
```

Thought I had fixed it everywhere but it seems that I have forgot a few places.

Reply:
done ; I also fixed the other file edit.py